### PR TITLE
Basic temporary frontend for the form submition

### PR DESCRIPTION
**Pull request by SerhiiCho**
I don't really know React so I did only basic alert message after we submit the form.
Added materialize css library to the head tags of the index.html document and removed comments, and added basic material markup to our form.

Check it out

### CODE REVIEW

**Comment by SerhiiCho** [comment]
Извини что на английском, мой набор текста на русском оставляет желать лучшего ))

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea
```
Comment (smpl) [inline .gitignore]: use global gitignore for ignore stuff of ide
that changes can be removed after global git ignore
that files not part of project

File: web/public/index.html [hunk 2]
```diff
@@ -1,38 +1,18 @@
...
+<head>
     <meta charset="utf-8" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
```
Comment (smpl) [inline web/public/index.html]: materialize can be loaded as npm package not use cdn

File: web/src/index.css [hunk 3]
```diff
@@ -1,14 +1,312 @@
...
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.mt-0,
+.my-0 {
+  margin-top: 0 !important;
 }
+
```
Comment (smpl) [inline web/src/index.css:10]: overwrite default style of materialize not looks great idea

Comment (SerhiiCho) [reply web/src/index.css:10]: Where did you see it?

Comment (SerhiiCho) [reply web/src/index.css:10]: It's just css helper classes for margin and padding.

**Review by smpl — APPROVED** [review]

File: web/src/index.css (hunk 3 above)
Comment (smpl) [reply web/src/index.css:10]: i think better to use something like a grid
and default indent

**Review by smpl — APPROVED** [review]

**Pull request merged**